lib/wanda/executions/evaluation.ex

```diff
@@ -242,7 +242,7 @@ defmodule Wanda.Executions.Evaluation do
            not errors?(agents_check_results) do
         :passing
       else
-        severity
+        eval_severity(severity, agents_check_results)
```

Comment:
`eval_` might be confused with an actual expression evaluation.

What about `result_with_severity`, `severity_based_result`, `with_severity` or something similar? 
Something that says the result is being calculated based also on severity.

Reply:
I'm thinking to put this piece of code even directly on the `aggregate_check_result` else side. It is pretty simple, so maybe we don't need an extra function.
What do you think?

Reply:
That's also a valid alternative :+1:

Reply:
Update amended